.github/workflows/ci.yml

```diff
@@ -68,27 +64,18 @@ jobs:
       - name: rustdoc
         run: RUSTDOCFLAGS="-Dwarnings" ./miri doc --document-private-items
 
-  # These jobs doesn't actually test anything, but they're only used to tell
-  # bors the build completed, as there is no practical way to detect when a
-  # workflow is successful listening to webhooks only.
-  #
-  # ALL THE PREVIOUS JOBS NEED TO BE ADDED TO THE `needs` SECTION OF THIS JOB!
-  end-success:
-    name: bors build finished
-    runs-on: ubuntu-latest
+  conclusion:
     needs: [build, style]
-    if: github.event.pusher.name == 'bors' && success()
-    steps:
-      - name: mark the job as a success
-        run: exit 0
-  end-failure:
-    name: bors build finished
+    # !cancelled() executes the job regardless of whether the previous jobs passed, failed or get skipped.
+    if: ${{ !cancelled() }}
```

Comment:
In https://github.com/rust-lang/miri-test-libstd/blob/master/.github/workflows/ci.yml we use `if: always()`, that seems a bit less odd?

Reply:
I now tested that it's actually not the same. `always()` will run the conclusion job even if the whole workflow is cancelled (which we rarely do manually). But if we cancel the workflow, we don't want the conclusion job to run. So `!cancelled()` is the better option, I think (yes, GHA statuses are confusing...).